test/controllers/dashboards_controller_test.rb

```diff
@@ -29,4 +29,20 @@ class DashboardsControllerTest < ActionDispatch::IntegrationTest
       end
     end
   end
+
```

Comment:
More test to come? These don't distinguish the cases afaik.

Reply:
lol busted, thank you for catching this. I'll add fails for now and flesh out the test cases later.